sql/clients_last_seen_raw_v1.sql

```diff
@@ -75,7 +59,11 @@ CREATE TEMP FUNCTION
           branch)
       WHERE
         prev IS NULL)));
-  --
+CREATE TEMP FUNCTION
+  udf_shift_one_day(x INT64) AS (IFNULL((x << 1) & udf_bitmask_lowest_28(),
```

Comment:
It looks like most usage of `udf_shift_one_day` has been replaced with `udf_shift_bits_one_day`, but there are still a few usages of the less-specifically-named one.

I'm pretty sure these two UDFs have the same definition and `udf_shift_one_day` is just a legacy name that happened to stay in use. Can we change all references so that `udf_shift_one_day` no longer appears anywhere?